.circleci/config.yml

```diff
@@ -155,6 +155,19 @@ jobs:
           cd frontend
           yarn typecheck
 
+  versioning:
+    docker:
+      - image: node:12
+    working_directory: ~/repo
+    steps:
+      - checkout
+      - run: |
+          yarn global add standard-version
+          standard-version release --no-verify
+      - run: |
+          yarn global add git
```

Comment:
I think we should test this new config first. Its fine to remove the filter for the versioning job and allow it to run on this branch first. Also, it should push to this branch too.

Here's a [guide](https://support.circleci.com/hc/en-us/articles/360018860473-How-to-push-a-commit-back-to-the-same-repository-as-part-of-the-CircleCI-job) to push back to GitHub. 

I've just added a user key to do the push. Just need to add the git email and name and then do the push.

You can also try run the job locally using the [circleci cli](https://circleci.com/docs/2.0/local-cli/) if you want.

Reply:
which git email and name should we use?

Reply:
Right hmmm I think we could use my email, chrsep@protonmail.com, Chrisando Pramudhita.